front/lib/api/assistant/configuration.ts

```diff
@@ -873,10 +876,13 @@ export async function createAgentConfiguration(
             name,
             description,
             instructions,
+            modelId: model.modelId,
+            providerId: model.providerId,
+            temperature: model.temperature,
             maxToolsUsePerRun: maxToolsUsePerRun,
             pictureUrl,
             workspaceId: owner.id,
-            generationConfigurationId: generation?.id || null,
+            generationConfigurationId: null,
```

Comment:
Shouldn't we keep the id until we phase out?

Reply:
Actually the column was removed from DB on this commit: https://github.com/dust-tt/dust/pull/4712/files#diff-766f5c1abd936709ce451f717525d138ff51a578da653997be29a431d9441b81L204-L225
But he forgot to edit both the type and the model.

I've cleaned things up on this commit: https://github.com/dust-tt/dust/pull/4723/commits/fd361080729b85cec4c5b57ccd5d85948e28d6a5